### PR TITLE
Mapped SkullEntityModel

### PR DESCRIPTION
**Pull request by Sollace**
SkullEntityModel.a -> skull
SkullEntityModel.setRotationAngles -> render

Also supplied parameter names for the constructor.

### CODE REVIEW

File: mappings/net/minecraft/client/render/entity/model/SkullEntityModel.mapping [hunk 1]
```diff
@@ -1,5 +1,11 @@
 CLASS dgf net/minecraft/client/render/entity/model/SkullEntityModel
-	METHOD a setRotationAngles (FFFFFF)V
+	FIELD a skull Ldhd;
+	METHOD <init> (IIII)V
+		ARG 1 textureU
+		ARG 2 textureV
+		ARG 3 textureWidth
+		ARG 4 textureHeight
+	METHOD a render (FFFFFF)V
```
Comment (modmuss50) [inline mappings/net/minecraft/client/render/entity/model/SkullEntityModel.mapping:8]: `renderWithAngles`? im not sure `render` is fully correct.

`setRotationAngles` is wrong though.

**Review by modmuss50 — CHANGES_REQUESTED** [review]
humm, im not sure on this, might be best to get a 2nd option on this.

File: mappings/net/minecraft/client/render/entity/model/SkullEntityModel.mapping (hunk 1 above)
Comment (Sollace) [reply mappings/net/minecraft/client/render/entity/model/SkullEntityModel.mapping:8]: I picked `render` for consistency with other model classes. Did think of maybe `orientAndRender` but ultimately decided against it.

I'm open to other options.

**Comment by asiekierka** [comment]
@kashike please review wrt MCPisms

**Comment by Sollace** [comment]
Why has this been marked as outdated?

**Comment by modmuss50** [comment]
it was made outdated due to us updating yarn to 1.14.3, the PR will need to be updated for that.

**Comment by Sollace** [comment]
Updated the PR. It's now based on the 1.14.3-pre1 branch.

**Comment by Sollace** [comment]
@modmuss50 Reminder, yo.

**Review by kashike — APPROVED** [review]

**Comment by Sollace** [comment]
Closed and merged but not actually merge? Well okay then...

**Comment by asiekierka** [comment]
GitHub won't mark it as "actually merged" unless you do something I'm not aware of... Not sure if there's a way to mark a manual merge as such - and I had to update this one to 1.14.3-pre2

**Comment by Sollace** [comment]
@asiekierka Weird. IIRC. I had to do a manual merge before, and used the commands github provided to do it. The site ended up correctly detecting the merge and updating the state of the PR.

Will see if I can find that the commands were. It was a combination of checking out the branch and merging it with `git merge --no-ff`

**Comment by Sollace** [comment]
@asiekierka Looks pretty easy, this is what github says for a pr merging a branch at stsyn:patch-1

image

**Pull request closed without merging**